config/secrets.yml

```diff
@@ -1,5 +1,7 @@
 common: &common
   secret_key_base: 'f0b774b4d06f7ea3357dec260cd6231367009d35525b3c437b27e299d52599bebd4eb1cd83fc3f69ff0d46a0510e5e4ba3bffd26b503b7713100f6083fc4baba'
+  auth_secret_key: '89e115653bae4fafda3fc4c45b64aa07e20ad4d75ab4c1637e85e44b50e4a5cd24ea4dc065e047642d2ed17bebae7e856a292ab1f46ff7d04fc473df1a9ac577'
```

Comment:
Yes, secrets are required for running in test and development. If you read this file you'll notice that in the production section we use values from ENV **exclusively**. This means that production WILL NOT fall back to these values (I included a comment `DO NOT use value in common as fallback` to make this clear in the code).

If these secrets are not set in the ENV and rails is running in production then (at least for `secret_key_base`) rails will raise an error and refuse to run. (I think it will run if `auth_secret_key` is missing since devise will fallback to using the value for `secret_key_base`. I also think it will run if `password_pepper` is missing since that is optional.)